src/Microsoft.DotNet.Darc/DarcLib/Helpers/DependencyFileManager.cs

```diff
@@ -60,6 +60,12 @@ public class DependencyFileManager : IDependencyFileManager
     private const string MaestroRepoSpecificBeginComment = "  Begin: Package sources from";
     private const string MaestroRepoSpecificEndComment = "  End: Package sources from";
 
+    private const string VersionDetailsPropsComment = """
+        This file is auto-generated by the Maestro dependency flow system.
+        Do not edit it manually, as it will get overwritten in the next flow.
+        This file should be imported into Version.props using "<Import Project="Version.Details.props" />"
+        """;
```

Comment:
```suggestion
    private const string VersionDetailsPropsComment = """
        This file is auto-generated by the Maestro dependency flow system.
        Do not edit it manually, as it will get overwritten by automation.
        This file should be imported by eng/Version.props.
        """;
```